README.md

```diff
@@ -1,2 +1,50 @@
 # PrabhakarAndAlex-Old-Apple.com-Clone
 This project is focused on learning gradient and background image positioning using CSS. In this project we're required to create clone of Old Apple.com's homepage.
+
+![screenshot](images/SiteScreenshot.png)
```

Comment:
fix the image link it's not showing on the readme file.

Reply:
Fixing in upcoming commit! Thank You!